### DISCUSSION TITLE
Api Credentials guidance

### DISCUSSION THREAD

**Discussion by Golpik-CDN** [AshleyRedman/WPClothes2Order#113] [Q&A] [answered]
I'm not sure what to put in these fields in the plugin 

1 - Order API Endpoint (required)
2 - Stock API Endpoint (required)

Kindly help me get started 
Thanks

**Comment by AshleyRedman** [comment, marked as answer]
Hi @Golpik-CDN, Clothes2Order has two URIs / endpoints for interacting with their API. 
Technical info [can be found here](https://www.clothes2order.com/docs/placing-an-order/).

So for:
"**1 - Order API Endpoint (required)**", please paste in the below:
`https://www.clothes2order.com/api/post-order/`

And for:
"**2 - Stock API Endpoint (required)**", please paste in the below:
`http://c2ostock.s3.amazonaws.com/products.csv
`

We made the choice to allow the plugin user to paste these in / update them over time s Clothes2Order may update these URIs at any time, so we would not need to make a new release each time. 

The above two URIs should work for you, however if not, I would get in touch with C2O them selves. 
I hope that answers your question & if so, can you please mark this as answered.

**Reply by Golpik-CDN** [reply]
Thank you much appreciated
